tests/manage/z_cluster/cluster_expansion/conftest.py

```diff
@@ -0,0 +1,73 @@
+import pytest
+import logging
+
+from ocs_ci.ocs import constants
+from ocs_ci.framework import config
+from ocs_ci.ocs.node import (
+    get_worker_nodes,
+    add_new_node_and_label_it,
+    add_new_node_and_label_upi,
+)
+from ocs_ci.ocs import machine as machine_utils
+
+log = logging.getLogger(__name__)
+
+
+@pytest.fixture()
+def add_nodes():
+    """
+    Test for adding worker nodes to the cluster while IOs
+    """
+
+    def factory(ocs_nodes=None, node_count=3):
+
```

Comment:
add some docstring please instead of this empty line

Reply:
Done